fe/fe-core/src/main/java/org/apache/doris/alter/RollupJobV2.java

```diff
@@ -494,6 +494,7 @@ private void onFinished(OlapTable tbl) {
             partition.visualiseShadowIndex(rollupIndexId, false);
         }
         tbl.rebuildFullSchema();
+        tbl.setState(OlapTableState.NORMAL);
```

Comment:
We can to batch rollup jobs, and I think the table's state should be set to NORMAL after all jobs are done,
which is in MaterializedViewHandler.onJobDone()